openslides_backend/models/models.py

```diff
@@ -253,6 +253,17 @@ class Meeting(Model):
     conference_open_microphone = fields.BooleanField(default=False)
     conference_open_video = fields.BooleanField(default=False)
     conference_auto_connect_next_speakers = fields.IntegerField(default=0)
+    conference_enable_helpdesk = fields.BooleanField(default=False)
```

Comment:
This file is autogenerated, as stated at the start. Please add these changes to the [`models.yml`](https://github.com/OpenSlides/OpenSlides/blob/openslides4-dev/docs/models.yml) and execute `make generate-models` after they are merged.

Reply:
See OpenSlides/OpenSlides#6153